litellm/proxy/example_config_yaml/otel_test_config.yaml

```diff
@@ -36,7 +36,9 @@ model_list:
 
 litellm_settings:
   cache: true
-  callbacks: ["otel", "prometheus"]
+  callbacks: ["otel"]
```

Comment:
this is intentional and this test was written because a specific user uses `callbacks: ["prometheus"]` 

Can you add a diff test that does `success_callback: ["prometheus"]  ` @krrishdholakia

Reply:
Oh! got it

Reply:
fixed - https://github.com/BerriAI/litellm/pull/6005/commits/36ad126c5dc8f39c61d33dd74f49cda8c01670a9